spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/HashingTFFeatureGenerator.java

```diff
@@ -79,11 +81,12 @@ public void initialize(SparkExecutionPluginContext context) throws Exception {
       return context.getSparkContext().emptyRDD();
     }
     outputSchema = outputSchema != null ? outputSchema : config.getOutputSchema(input.first().getSchema());
-    final Map<String, String> mapping = config.getFeatureListMapping();
+    final Map<String, String> mapping = SparkUtils.getFeatureListMapping(config.outputColumnMapping);
```

Comment:
Sorry if I was unclear about this, did not mean to create a utility function to parse the string. It is fine to have the parse method in the individual plugin configs, since you need to give a plugin-specific error message. Just meant you should be passing the Map into the validation method instead of the string.